src/components/ApplicationDashboard.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react'
+import styled from 'styled-components'
+import { H1 } from './Typography'
+import { Button } from './Input/Button'
+import { SOCIAL_LINKS } from '../utility/Constants'
+import facebook from '../assets/icons/facebook.svg'
+import instagram from '../assets/icons/instagram.svg'
+import medium from '../assets/icons/medium.svg'
+import twitter from '../assets/icons/twitter.svg'
+import { ReactComponent as HandWave } from '../assets/hand-wave.svg'
+
+const Container = styled.div`
+  margin: 5em auto;
+  width: 70%;
+`
+
+const WelcomeHeader = styled.div`
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  margin-bottom: 1em;
+`
+
+const WelcomeMessage = styled(H1)`
+  text-align: center;
+`
+
+const StyledHandWave = styled(HandWave)`
+  margin-left: 10px;
+  margin-top: 20px;
+`
+
+const AppLinks = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`
+
+const HackerAppText = styled.p`
+  color: ${p => p.theme.colors.primary};
+  margin-top: 1.55em;
+`
+
+const EditAppButton = styled(Button)`
+  width: 250px;
+  margin-right: 0;
+`
+
+const StatusContainer = styled.div`
+  min-height: 350px;
+  padding: 2.5em 4.5em 2.5em 3em;
+  border-radius: 21px;
+  background-color: ${p => p.theme.colors.applicationCard};
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+`
+
+const AppStatusText = styled.p`
+  font-size: 1.25em;
+  font-weight: 700;
+  margin-top: 0;
+`
+
+const StatusBlurbText = styled.p`
+  color: ${p => p.theme.colors.primary};
+  font-weight: 700;
+  line-height: 2em;
+  margin-top: 0.75em;
+`
+
+const FooterContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-left: -24px;
+`
+
+const SocialMediaIcons = styled.img`
+  margin: 24px;
```

Comment:
<img width="228" alt="Screen Shot 2020-12-01 at 1 13 57 PM" src="https://user-images.githubusercontent.com/38872354/100798189-1f89a900-33d8-11eb-860e-469b180035f3.png">
These are pretty much vertical on mobile, let's shrink margin to 8px; and the size of the icons to 24px; in mobile

```suggestion
  margin: 24px;
  ${maxWidthMediaQueries('mobile')} {
width: 24px;
height: 24px;
  margin: 16px;
  }
```